cypress/component/components/NavBar.cy.js

```diff
@@ -0,0 +1,64 @@
+import NavBar from '../../../src/components/NavBar.vue';
+
+const staticResponse = {
+  fields: {
+    claims: {
+      mapValue: {
+        fields: {
+          adminUid: { stringValue: 'yXuZ8S0En1UsOE4C0uh6wUlQ5Wt1' },
+          assessmentUid: { stringValue: 'FW0G9oXYQjMpwPdyptVJOsbb6J42' },
+          roarUid: { stringValue: 'yXuZ8S0En1UsOE4C0uh6wUlQ5Wt1' },
+          someOtherField: { stringValue: 'someOtherValue' },
+        },
+      },
+    },
+  },
+};
+
+describe('<NavBar />', () => {
+  beforeEach(() => {
+    cy.setAuthStore().as('authStore');
+
+    // Intercept network calls and respond with mock data
+    cy.intercept(
+      'GET',
+      'https://firestore.googleapis.com/v1/projects/gse-roar-admin-dev/databases/(default)/documents/userClaims/**/*',
+      (req) => {
+        req.reply({
+          statusCode: 200,
+          body: staticResponse,
+          delay: 1000,
+        });
+      },
+    ).as('userClaims');
+  });
+
+  it('mounts using default Cypress viewport', () => {
+    cy.mount(NavBar);
+
+    // Await the intercepted request to check that it was successful
+    cy.wait('@userClaims').then((interception) => {
+      if (interception?.response?.statusCode === 200) {
+        expect(interception?.response.body).to.deep.equal(staticResponse);
+        cy.log('Interception successful', interception?.response);
```

Comment:
```suggestion
```

Technically don't need to log this as Cypress already does that for you, exposing request payload and response in the Cypress UI. But if you want to keep this for the CI logs, feel free. 
<img width="2453" alt="Screenshot 2024-09-05 at 20 34 14" src="https://github.com/user-attachments/assets/0b2b7bc4-4e6a-4200-a956-cdc672a4f806">